docs/content/latest/benchmark/sysbench-ysql.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Benchmark YSQL performance using Sysbench
+headerTitle: Sysbench
+linkTitle: Sysbench
+description: Benchmark YugabyteDB YSQL performance using Sysbench.
+headcontent: Benchmark YugabyteDB YSQL performance using Sysbench.
+menu:
+  latest:
+    identifier: sysbench-ysql
+    parent: benchmark
+    weight: 5
+aliases:
+  - /benchmark/sysbench/
+showAsideToc: true
+isTocNested: true
+---
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+
+  <li >
+    <a href="/latest/benchmark/sysbench-ysql/" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL
+    </a>
+  </li>
+
+</ul>
+
+Sysbench is a popular tool for benchmarking databases like Postgres and MySQL, as well as system capabilities like CPU, memory and I/O.
+Follow the steps below to run Sysbench against YugabyteDB.
+The version we are using is forked from the [official](https://github.com/akopytov/sysbench) version with a few modifications to better reflect YugabyteDB's distributed nature.
+
+## Step 1. Install Sysbench.
+
+You can do this by running the following commands.
+
+```sh
+$ cd $HOME
+$ git clone https://github.com/yugabyte/sysbench.git
+$ cd sysbench
+$ ./autogen.sh && ./configure --with-pgsql && make -j && sudo make install
+```
+
+{{< note title="Note" >}}
+The above steps will install the sysbench utility in '/usr/local/bin'
+{{< /note >}}
+
+## Step 2. Start YugabyteDB
+
+Start your YugabyteDB cluster by following the steps in [Quick start](https://docs.yugabyte.com/latest/quick-start/explore-ysql/).
+
+## Step 3. Run all OLTP workloads
+
+There is a handly shell script 'run_sysbench.sh' that loads the data and runs the various workloads.
+```sh
+./run_sysbench.sh
```

Comment:
Please add a note on how to specify IP address for this script.

Reply:
Done.